components/datadog/fakeintake/connectionExporter.go

```diff
@@ -9,37 +9,37 @@ import (
 )
 
 const (
-	stackKey = "fakeintake-url"
+	stackKey = "fakeintake-ip-address"
 )
 
 // ConnectionExporter contains pulumi side data and the export key
 type ConnectionExporter struct {
-	URL      pulumi.StringInput
-	stackKey string
+	IPAddress pulumi.StringInput
```

Comment:
I'd say `host`, it could be a DNS name.

Reply:
Agreed !
I’ve renamed `IPAddress` into `Host` in https://github.com/DataDog/test-infra-definitions/pull/133/commits/30b5719537cfb2b960ccdc81bdd2b7f0ce971ff8.